e2e/js/tsconfig.spec.json

```diff
@@ -15,6 +15,7 @@
     "**/*.spec.jsx",
     "**/*.test.jsx",
     "**/*.d.ts",
-    "jest.config.ts"
+    "jest.config.ts",
+    "local-registry-setup.ts"
```

Comment:
I don't think this is necessary?

followup: can you delete it?

Reply:
deleted